charts/timed/templates/deployment-backend.yaml

```
@@ -25,6 +25,11 @@ spec:
                 secretKeyRef:
                   name: {{ include "timed.postgresql.fullname" . }}
                   key: postgresql-password
+            - name: DJANGO_AUTH_LDAP_BIND_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "timed.fullname" . }}-backend
```

The secret referenced here is missing. There is a secret called `{{ include "timed.fullname" . }}-django` where we could add the `ldap-apssword` key.

```suggestion
                  name: {{ include "timed.fullname" . }}-django
```

It look like that secret in `templates/pre-install-secret-backend.yaml` is a pre-install hook to ensure we only generate `DJANGO_SECRET_KEY` once, but I'm not sure that really makes sense. We could probably consider changing it to a regular secret and just have folks explicitly set `backend.service.secret` if they do not want to rotate the secret on a regular basis.